tests/conftest.py

```diff
@@ -64,6 +65,16 @@ def pytest_addoption(parser):
                      help="Nebula Graph workspace")
 
 
+def pytest_bdd_step_error(request, feature, scenario, step, step_func, step_func_args):
```

Comment:
`pytest_bdd_step_error` not by called?

Reply:
it registered by pytest automatically.
1. pytest scan `conftest`
2. match the same definition with hook in pytest-bdd.

https://github.com/pytest-dev/pytest-bdd/blob/master/pytest_bdd/plugin.py

Reply:
👍